.github/workflows/update-purls.yml

```diff
@@ -0,0 +1,52 @@
+name: Update PURLs list of dependencies for dependabot
+
+on:
+  pull_request:
+  workflow_dispatch:
+
+permissions:
+  pull-requests: write
+  contents: write
+  repository-projects: read
+jobs:
+  update-purls:
+    name: Update PURLs list of dependencies
+    runs-on: ubuntu-latest
+    steps:
+    - uses: GitHubSecurityLab/actions-permissions/monitor@v1
+      with:
+        config: ${{ vars.PERMISSIONS_CONFIG }}
+    - name: Generate GitHub App Token
+      id: generate_token
+      uses: actions/create-github-app-token@v2x
+      with:
+        app-id: ${{ secrets.AKO_RELEASER_APP_ID }}
+        private-key: ${{ secrets.AKO_RELEASER_RSA_KEY }}
+        owner: ${{ github.repository_owner }}
+        repositories: mongodb-atlas-cli-plugin-kubernetes
+    - name: Checkout code
+      uses: actions/checkout@v4.1.1
+      with:
+        ref: ${{ github.head_ref }}
+        token: ${{ steps.app-token.outputs.token }}
```

Comment:
GitHub does not really like that `app-token`, did you mean `generate_token`instead?